src/libraries/System.Net.Security/tests/FunctionalTests/SslStreamNetworkStreamTest.cs

```diff
@@ -900,59 +1034,50 @@ public async Task SslStream_ClientCertificate_SendsChain()
 
             Assert.NotEqual(0, retries);
 
-            var clientOptions = new SslClientAuthenticationOptions() { TargetHost = "localhost" };
+            var clientOptions = new SslClientAuthenticationOptions()
+            {
+                TargetHost = "localhost",
+                EnabledSslProtocols = SslProtocols.Tls12
```

Comment:
Is there reason to fix it to Tls 1.2? This will become obsolete at some point....

Reply:
no, it is a leftover from debugging (I was checking the certs sent over the wire)